apps/src/lib/kits/maker/ui/DiscountCodeInstructions.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {Component, PropTypes} from 'react';
+import i18n from "@cdo/locale";
+import View_Cart from "../../../../../../dashboard/app/assets/images/maker/View_Cart.png";
+import Add_to_cart from "../../../../../../dashboard/app/assets/images/maker/Add_to_cart.png";
+import Enter_Discount_Code from "../../../../../../dashboard/app/assets/images/maker/Enter_Discount_Code.png";
```

Comment:
My first reaction is that these images should probably live in apps somewhere rather than dashboard, though I'm not sure that's actually true. I'm actually a little surprised this would work (importing from a dashboard path). Was there some other place doing something like this that this was based off of?

Reply:
I would also name these variables something like `viewCart` rather than `View_Cart`

Reply:
I just wasn't sure where to put the images. I've added them to apps/static/maker now.